docs/components/MainSectionCard.js

```diff
@@ -15,6 +15,7 @@ type Props = {|
   defaultCode?: string,
   description?: string,
   iframeContent?: string,
+  shadeColor?: 'tertiary' | 'darkWash' | 'lightWash' | 'default',
   shaded?: boolean,
```

Comment:
leaving shaded for backwards compatibility and ease-of-use, shadeColor is like a power-up option